src/Components/OpenModelControl.jsx

```diff
@@ -35,13 +38,13 @@ function OpenModelDialog({isDialogDisplayed, setIsDialogDisplayed, fileOpen}) {
   const classes = useStyles()
   return (
     <Dialog
-      icon={<OpenIcon/>}
+      icon={<ModelsIcon/>}
       headerText='Model Access'
       isDialogDisplayed={isDialogDisplayed}
       setIsDialogDisplayed={setIsDialogDisplayed}
       content={
         <div className={classes.content}>
-          <LocalFileOpen/>
+          <GitHubIcon style = {{width: '50px', height: '50px'}}/>
```

Comment:
Need to use inline styles here.
We are styling all of the svgs in the dialog which over writes the class styles.